ios/PurchasesHybridCommon/PurchasesHybridCommonUI/Paywalls/PaywallProxy.swift

```diff
@@ -182,6 +182,7 @@ import UIKit
 
         controller.delegate = self
         controller.modalPresentationStyle = .pageSheet
+        controller.view.backgroundColor = .systemBackground
```

Comment:
Ohhh that explains it! I think that makes sense... I do wonder if this could affect any paywalls if no background is set... But maybe this would only affect the loading paywall?

Reply:
Yes, this also fixes it for any paywalls. But I think that's the desired behavior